x/distribution/client/testutil/cli_test.go

```diff
@@ -19,3 +20,9 @@ func TestIntegrationTestSuite(t *testing.T) {
 func TestGRPCQueryTestSuite(t *testing.T) {
 	suite.Run(t, new(GRPCQueryTestSuite))
 }
+
+func TestWithdrawAllSuite(t *testing.T) {
```

Comment:
Added a new suite here, since there are few tests in existing integration suite keep on failing (seems like non-determinism).